### PR TITLE
Fixed up LoginManagerUT, Login Manager, UsersController for login

### PR DESCRIPTION
**Pull request by cf2080**
# Issue/Ticket

CLOSES #XX

# Description
Added parameters in Login Manager class due to Unit Test results. Unit Tests are now passing for Login ManagerUT

## Type of change
Chore

# How Has This Been Tested?
It has been Unit Tested
- LoginManagerUT

Put any notes here

### CODE REVIEW

File: Backend/ManagerLayer/Login/LoginManager.cs [hunk 1]
```diff
@@ -26,11 +26,11 @@ public LoginManager()
             _tokenService = new TokenService();
         }
 
-        public bool LoginCheckUserExists(string email)
+        public bool LoginCheckUserExists(LoginRequest request)
         {
             using (var _db = new DatabaseContext())
             {
```
Comment (alfredovargasjr) [inline Backend/ManagerLayer/Login/LoginManager.cs:33]: DB access should be put i the route layer for transactions

File: Backend/ManagerLayer/Login/LoginManager.cs [hunk 2]
```diff
@@ -58,13 +59,17 @@ public bool LoginCheckUserDisabled()
...
-                string hashedPassword = _passwordService.HashPassword(password, user.PasswordSalt);
+                user = _userService.GetUser(_db, request.email);
+                string hashedPassword = _passwordService.HashPassword(request.password, user.PasswordSalt);
                 if (userRepo.ValidatePassword(user, hashedPassword))
                 {
+                    user.IncorrectPasswordCount = 0;
+                    _userService.UpdateUser(_db, user);
+                    _db.SaveChanges();
```
Comment (alfredovargasjr) [inline Backend/ManagerLayer/Login/LoginManager.cs]: When calling .savechanges(), catch any errors that can be thrown, and when caught, rollback changes, you can look at register route.

**Review by alfredovargasjr — DISMISSED** [review]

File: Backend/ManagerLayer/Login/LoginManager.cs (hunk 1 above)
Comment (cf2080) [reply Backend/ManagerLayer/Login/LoginManager.cs]: if I do that a lot of the logic will move in the controller and make the method will be bigger, wouldn't it? @alfredovargasjr

**Review by julianpoy — CHANGES_REQUESTED** [review]
```
.vs/LoginFrontend/v15/.suo
.vs/ProjectSettings.json
.vs/slnx.sqlite
```

Should all be removed

**Comment by alfredovargasjr** [comment]
@cf2080 get the latest changes from master

**Review by julianpoy — APPROVED** [review]

**Pull request merged**